runtimes/peregrine/src/system/proxy.rs

```diff
@@ -205,7 +217,20 @@ impl InstanceFilter<RuntimeCall> for ProxyType {
 							| pallet_web3_names::Call::unban { .. }
 							| pallet_web3_names::Call::update_deposit { .. }
 							| pallet_web3_names::Call::change_deposit_owner { .. }
-					),
+					)
+					| RuntimeCall::BondedCurrencies(
+						// Excludes `finish_destroy`
```

Comment:
that's pretty pointless, the call is permissionless

Reply:
-.- true.

Reply:
I allow all tx: [f619872](https://github.com/KILTprotocol/kilt-node/pull/818/commits/f6198720463b18e6a82c596d035323c470432099)